tools/oas-apigee-mock/lib/commands/generateApi/generateProxy.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var builder = require('xmlbuilder')
+var fs = require('fs')
+var path = require('path')
+
+module.exports = function generateProxy (apiProxy, options, api, cb) {
+  var destination = options.destination || path.join(__dirname, '../../../api_bundles')
+  if (destination.substr(-1) === '/') {
+    destination = destination.substr(0, destination.length - 1)
+  }
+  var rootDirectory = destination + '/' + apiProxy + '/apiproxy'
+  var root = builder.create('APIProxy')
+  root.att('revison', 1)
```

Comment:
I am pretty sure this attribute is optional. We can remove it as it will be set on deployment.